localstack/services/ssm/ssm_listener.py

```diff
@@ -64,6 +71,43 @@ def get_params_and_secrets(names):
     return result
 
 
+def get_params_by_path_with_labels(path='', param_filters=None, labels_to_filter=None, recursive=False):
+    ssm_client = aws_stack.connect_to_service('ssm')
+    result = {'Parameters': []}
+    filters = [{'Key': 'Path', 'Values': [path]}]
+    filters.extend(param_filters)
+    if recursive:
+        filters[0]['Option'] = 'Recursive'
+
+    def filter_by_label(param=None, labels=None):
```

Comment:
nit: I guess this function signature could be simplified to:
```
def filter_by_label(param, labels):
    ...
```
(as `param['Labels']` would fail anyway, in case `param` is not passed to the function)